src/genotyper.cc

```diff
@@ -272,21 +255,19 @@ static Status translate_genotypes(const genotyper_config& cfg, const unified_sit
 
     // reference allele maps if it contains the unified site
     range rng(record);
-    if (rng.contains(site.pos)) {
+    if (rng.overlaps(site.pos)) {
```

Comment:
I think we can now unconditionally set `allele_mapping[0] = 0` because subsequent changes now ensure we're looking at a rep record if we get to this point, right?